### DISCUSSION TITLE
deploy the model on huggingface

### DISCUSSION THREAD

**Discussion by serrysibaee** [karpathy/build-nanogpt#58] [Q&A]
Hello Mr. Karpathy and everyone I hope you are doing well.
we trained small models with the code and tried to upload the model to huggingface using this article [https://huggingface.co/docs/hub/en/models-uploading#upload-a-pytorch-model-using-huggingfacehub](url) 
but the pipeline start giving problems with transposes and configurations 

my question is did any one tried to upload the model on huggingface and how did he do it?

**Comment by joHussien** [comment]
@serrysibaee Have you solved this?
